host/lib/usrp/common/lmx2592.cpp

```diff
@@ -519,6 +521,54 @@ class lmx2592_impl : public lmx2592_iface
         }
     }
 
+    void _set_fcal_adj_values(const double pfd_freq)
+    {
+        // Adjust FCAL speed for particularly high or low PFD frequencies
+        if(pfd_freq < 5e6)
```

Comment:
This will not work with our clang-format settings (but we can fix that).

Reply:
Ah, is it the brace wrapping on the conditional statement?  Sorry about that - let me know if I can do anything on my end to help at this point.

Reply:
Don't worry, we have tools that do that for us. That was just an FYI. Your PRs have been extremely well crafted so far, down to the formatting of the commit message :).